packages/babel-standalone/test/helpers/env.js

```diff
@@ -0,0 +1,22 @@
+export function setup() {
+  // chalk, which we bundle in @babel/standalone, relies on `window.navigator`
+  // for the browser build:
+  // https://github.com/chalk/chalk/blob/f399cd0ff69841e88cca89d43a49f1cc9ba2efd5/source/vendor/supports-color/browser.js#L4
+  // We only bundle Chalk 5 in local dev and in Babel 8, so we avoid this
+  // "polyfill" when releasing Babel 7 to make sure that we do not accidentally
+  // bundle Chalk 5.
+
+  if (process.env.BABEL_8_BREAKING) {
+    globalThis.navigator = {};
+  } else if (!process.env.IS_PUBLISH) {
+    global.navigator = {};
+  }
+}
```

Comment:
Can we replace it with `navigator || {}` in rollup?
I remember someone seems to be using `@babel/standalone` in node, maybe it's not worth breaking them.
If you agree I can try to do this. :)

Reply:
Yes please!